src/test/developTest/resources/wdr/wdr_sendung_overview.html

```diff
@@ -0,0 +1,984 @@
+<div class="section sectionA hidden" >
+<div class="breadcrumb screenReader">
+<p class="hidden">Sie befinden sich hier: </p>
+<ul>
+<li>
+<a href="/index.html">WDR</a>
+</li>
+<li>
+<a href="/mediathek/index.html">Mediathek</a>
+</li>
+<li>
+<a href="/mediathek/video/index.html">Video</a>
+</li>
+<li>
+<a href="/mediathek/video/sendungen/index.html">Sendungen A-Z</a>
+</li>
+<li>
+Ausgerechnet
+</li>
+</ul>
+</div>
+</div>
+<a name="goToContent" class="hidden"></a>
+
+<div class="section sectionA" >
+<div class="con tabNaviCon clean">
+<h2 class="headline hidden">Auswahl Mediathek</h2>
+<div class="tabNavi">
+<ul>
+<li class="video active">
+<a href="/mediathek/video/index.html">
+<span>Videos</span>
+</a>
+<div data-ctrl-collapsible="{'id':'navProgram','action':{'default':['openEntries','disable'], 'xs':['closeEntries','closeEntriesOnVanish','enable'],'s':['closeEntries','closeEntriesOnVanish','enable']}}" class="innerNavi videos">
+<div role="search" data-ctrl-navprogram-entry="{'id':'wdr-suche-video'}" class="searchBox">
+<div data-ctrl-navprogram-wdr-suche-video-trigger="{'action':{'default':['closeOtherEntries','toggleBody']}}" class="searchToggle"><a href="/mediathek/video/index.html">Suche</a></div>
+<form name="form_wdr-suche-video" method="post" action="/mediathek/video/suche/index.jsp" data-ctrl-navprogram-wdr-suche-video-body="{}" data-ctrl-formular="{'id':'form_wdr-suche-video'}" enctype="application/x-www-form-urlencoded" class="searchForm"/>
+<input name="q" type="text" value="Videos in der Mediathek suchen" data-ctrl-form_wdr-suche-video-input="{'action':{'default':['switchValue']}}" class="searchInput"/>
+<input type="hidden" name="pt_video" value="on"/>
+<a href="/mediathek/video/index.html" data-ctrl-form_wdr-suche-video-trigger="{'action':{'default':['submit']}}" class="searchSubmit"> </a>
+</form>
+</div>
+<div class="naviSerialProgram" role="navigation" data-ctrl-navprogram-entry="{'id':'nav1'}" >
+<div class="menueToggle hasArrow" data-ctrl-navprogram-nav1-trigger="{'action':{'default':['closeOtherEntries','toggleBody']}}" style="position: static; cursor: default;">
+<a href="#">Unterpunkte auf-/zuklappen</a>
+</div>
+<ul class="menu" data-ctrl-navprogram-nav1-body="{}">
+<li >
+<span class="collapsed subressort" >
+<a href="/mediathek/video/online-first/index.html">Vorab ansehen</a>
+</span>
+</li>
+<li >
+<span class="collapsed subressort" >
+<a href="/mediathek/video/live/index.html">Livestream</a>
+</span>
+</li>
+<li >
+<span class="collapsed subressort" >
+<a href="/mediathek/video/sendungverpasst/index.html">Sendung verpasst</a>
+</span>
+</li>
+<li >
+<span class="collapsed subressort" >
+<a href="/mediathek/video/sendungen-a-z/index.html">Sendungen A-Z</a>
+</span>
+</li>
+</ul>
+<div class="clearMe"></div>
+</div>
+</div>
+</li>
+<li class="audio">
+<a href="/mediathek/audio/index.html">
+<span>Audios</span>
+</a>
+</li>
+</ul>
+</div>
+</div>
+</div>
+<!-- wdr -->
+
+<a name="goToContent" class="hidden"></a>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "name" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AS1827)